engines/billing/questions.md

```diff
@@ -0,0 +1,22 @@
+Questions
+====================
+
+1. Should admin be able to see subscriptions?
```

Comment:
There is something called 'Past Due' on subscriptions status even though Braintree will automatically attempt to charge past due subscriptions at the beginning of each new billing cycle. As it is a status, it can be reached by admin and it is also shown in the dashboard of the account. See here: https://developers.braintreepayments.com/ios+ruby/guides/recurring-billing/overview#past-due. It can be done but it depends on the retry and on the Control Panel.